ee/clickhouse/queries/clickhouse_funnel.py

```diff
@@ -102,7 +102,7 @@ def run(self, *args, **kwargs) -> List[Dict[str, Any]]:
         res = []
         for result_tuple in results:
             result = list(result_tuple)
-            person = Person(pk=result[0])
+            person = Person(pk=result[0], uuid=result[0])
```

Comment:
how is this used later? this seems to be begging for type incompatibility

Reply:
Hm not sure I understand the question. It's used in queries/funnel.py in data_to_return. We only use the uuid now. How do you think we could improve this?

Reply:
ah nvm